src/components/ComputeModal.js

```diff
@@ -149,7 +149,7 @@ const SparkInterface = ({ sparkInterface, namespace, name, onDismiss }) => {
               style: { marginRight: 'auto' },
               onClick: onDismiss,
               ...Utils.newTabLinkProps
-            }, ['Launch', icon('pop-out', { size: 12, style: { marginLeft: '0.5rem' } })])
+            }, ['Open', icon('pop-out', { size: 12, style: { marginLeft: '0.5rem' } })])
```

Comment:
Can you add a screenshot of where this is? I can't figure it out.

Reply:
Its for the spark interface associated with dataproc clusters (you need to select something other than the default for a jupyter environment in this drop down, and then click `Manage and monitor spark console`
<img width="539" alt="image" src="https://user-images.githubusercontent.com/6465084/161107174-50d2f1b4-287a-45f2-bc3e-09439eddb644.png">

Here's what it looks like:
<img width="545" alt="image" src="https://user-images.githubusercontent.com/6465084/161107067-a265c951-c535-4c65-864a-7253e07ab5bd.png">

Reply:
Thanks!